src/Components/Category/Category.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { CategoryCard } from "./CategoryCard";
+
+export const Category = () => {
+  const [category, setCategory] = useState([]);
+  useEffect(() => {
+    (async () => {
+      const response = await axios.get("/api/categories");
+      setCategory(response.data.categories);
+    })();
+  }, []);
```

Comment:
do api calls in try catch blocks for error handlining

Reply:
Fixed, Thanks.